src/pages/home_page/sub_pages/Gallery.tsx

```diff
@@ -112,22 +111,16 @@ const Gallery = (): JSX.Element => {
                 zIndex: 2,
             }}
         >
-            <ImageGrid
-                img_one={one}
-                img_two={two}
-                img_three={three}
-                img_four={four}
-                img_five={five}
-                img_six={six}
-            />
-            <ImageGrid
-                img_one={seven}
-                img_two={eight}
-                img_three={nine}
-                img_four={ten}
-                img_five={eleven}
-                img_six={twelve}
+           {imgData.map((index:any,item:any)=>{
```

Comment:
1. Give proper types to `index` and `item`. Remove `item` from here if not necessary.
2. Format.
3. The logic is entirely wrong. Did you run this locally and look at the gallery it produces?